components/expenses/ExpenseFormPayeeStep.js

```diff
@@ -371,29 +363,29 @@ const ExpenseFormPayeeStep = ({
                   )}
                 </FastField>
               )}
-              <FastField name="invoiceInfo">
-                {({ field }) => (
-                  <StyledInputField
-                    name={field.name}
-                    label={formatMessage(msg.invoiceInfo)}
-                    labelFontSize="13px"
-                    required={false}
-                    mt={3}
-                  >
-                    {inputProps => (
-                      <Field
-                        as={StyledTextarea}
-                        {...inputProps}
-                        {...field}
-                        minHeight={80}
-                        placeholder={formatMessage(msg.invoiceInfoPlaceholder)}
-                      />
-                    )}
-                  </StyledInputField>
-                )}
-              </FastField>
             </Fragment>
           )}
+          <FastField name="invoiceInfo">
```

Comment:
While that was not very clear given the previous condition, this field was in the above condition for a reason: we only want to show this field when we're going to generate an actual PDF invoice. We should wrap it in a:

```jsx
{expense.type === expenseTypes.INVOICE && (
  ...
})
```